packages/dds/tree/src/feature-libraries/default-schema/defaultEditBuilder.ts

```diff
@@ -435,7 +435,7 @@ export interface OptionalFieldEditBuilder<TContent> {
 
 /**
  */
```

Comment:
Any docs we can add to this? Alternatively, can we remove the blank comment block?